BibBuddy/app/src/main/java/de/bibbuddy/BookModel.java

```diff
@@ -172,4 +173,53 @@ public Author getAuthorById(Long id) {
     return authorDao.findById(id);
   }
 
+  private void sortBookList(SortCriteria sortCriteria) {
+    switch (sortCriteria) {
+      case NAME_ASCENDING:
+        bookList.sort(new SortName());
+        break;
+
+      case NAME_DESCENDING:
+        bookList.sort(new SortName().reversed());
+        break;
+
+      case MOD_DATE_OLDEST:
+        bookList.sort(new SortDate());
+        break;
+
+      case MOD_DATE_LATEST:
+        bookList.sort(new SortDate().reversed());
+        break;
+
+      default:
+        break;
+    }
+  }
+
+  /**
+   * Gets the sorted bookList by sortCriteria.
+   *
+   * @param sortCriteria sortCriteria of the list
+   * @return Returns the sorted bookList
+   */
+  public List<BookItem> getSortedBookList(SortCriteria sortCriteria) {
+    sortBookList(sortCriteria);
+
+    return bookList;
+  }
```

Comment:
Why here without booklist as parameter? 
Possibly add explanation for the two methods.

Reply:
Because the bookList in the BookModel class only needs to be sorted and the bookList is already up to date.